src/simulate.jl

```diff
@@ -0,0 +1,89 @@
+using DifferentialEquations
+const DE = DifferentialEquations
+
+function simulate(cp::AbstractNeuralNetworkControlProblem, args...; kwargs...)
+    ivp = plant(cp)
+    network = controller(cp)
+    st_vars = state_vars(cp)
+    n = length(st_vars)
+    X₀ = LazySets.Projection(initial_state(ivp), st_vars)
```

Comment:
```suggestion
    X₀ = Projection(initial_state(ivp), st_vars)
```
(i think this is no longer needed)